src/electos/ballotmaker/ballots/contest_data.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass, field
+from typing import List
```

Comment:
This isn't being used. However if/when type annotations are added to the dataclass fields, it will be needed for Python 3.8 compatibility. The use of e.g. `list[str]` instead of `typing.List[str]` doesn't show up until Python 3.9. It's basically your call whether it should stick around until annotations are added or not.

Reply:
Removed for simplicity